dev/skinny-requirements.txt

```diff
@@ -1,5 +1,3 @@
-## Small test reqs
```

Comment:
Speaking of test requirements, is there a way we can be sure that installing `sqlalchemy` / `sqlparse` / `alembic` in our skinny client test environment (https://github.com/mlflow/mlflow/blob/032ec8841dad13492a6eb3c9acdf78803ed90f76/dev/run-python-skinny-tests.sh#L17) doesn't also install pandas and numpy? It'd be a bit of a bummer if numpy / pandas were somehow pulled in and we ended up missing pd / np imports in skinny client modules (tracking, etc)

Reply:
I split test_skinny.py into test_skinny and test_skinny_sql. 

Test_skinny_sql runs before the pip installs and tests_skinny, checks pandas, after the pip installs

Reply:
Were those changes pushed?